iphone/Classes/TiUIWebView.m

```diff
@@ -712,6 +722,20 @@ - (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)
 	}
 
 	NSURL * newUrl = [request URL];
+    
+    if (blacklistedURLs && blacklistedURLs.count > 0) {
+        NSString *urlAbsStr = [newUrl absoluteString];
+        
+        for (NSString *blackListedUrl in blacklistedURLs) {
+            if([urlAbsStr rangeOfString:blackListedUrl options:NSCaseInsensitiveSearch].location !=NSNotFound) {
+                if([self.proxy _hasListeners:@"onStopBlacklistedUrl"]) {
+                    NSDictionary *eventDict = [NSDictionary dictionaryWithObjectsAndKeys:urlAbsStr,@"url",@"Webview did not load blacklisted url.", @"messsage", nil];
+                    [self.proxy fireEvent:@"onStopBlacklistedUrl" withObject:eventDict];
```

Comment:
Try to use `[self proxy]` instead of `self.proxy`, but just out of personal preference :-)